### PR TITLE
Added docker installation guide

### PR DESCRIPTION
**Pull request by eorituz**
@czepluch please review.

### CODE REVIEW

File: docs/overview_and_guide.rst [hunk 1]
```diff
@@ -44,6 +44,23 @@ Or you can use Homebrew to install the most up to date binary::
 An Ethereum client is required in both cases. The Raiden binary takes the same command line
 arguments as the ``raiden`` script.
 
+Docker
+******
+
+ There are two options to run a raiden docker image:
+
+ Create the Image yourself and use our `Dockerfile <https://github.com/raiden-network/raiden/blob/master/docker/Dockerfile>`_ as template or use the already build image from Dockerhub::
```
Comment (LefterisJP) [inline docs/overview_and_guide.rst]: /s/build/built

File: docs/overview_and_guide.rst [hunk 2]
```diff
@@ -44,6 +44,21 @@ Or you can use Homebrew to install the most up to date binary::
 An Ethereum client is required in both cases. The Raiden binary takes the same command line
 arguments as the ``raiden`` script.
 
+	Docker
```
Comment (czepluch) [inline docs/overview_and_guide.rst]: No need to indent here. I know - very nitpicky, but let's stay consistent.

File: docs/overview_and_guide.rst [hunk 3]
```diff
@@ -44,6 +44,21 @@ Or you can use Homebrew to install the most up to date binary::
...
+******
+
+  There are two options to run a raiden docker image:
```
Comment (czepluch) [inline docs/overview_and_guide.rst]: Why indent here?

File: docs/overview_and_guide.rst [hunk 4]
```diff
@@ -44,6 +44,21 @@ Or you can use Homebrew to install the most up to date binary::
...
+
+  Create the Image yourself and use our `Dockerfile <https://github.com/raiden-network/raiden/blob/master/docker/Dockerfile>`_ as template or use the already built image from Dockerhub::
```
Comment (czepluch) [inline docs/overview_and_guide.rst]: Same as above.

File: docs/overview_and_guide.rst [hunk 5]
```diff
@@ -44,6 +44,21 @@ Or you can use Homebrew to install the most up to date binary::
...
+
+      docker run -it raidennetwork/raiden:latest
+
+  The required keystore can easily be mounted in the docker container::
```
Comment (czepluch) [inline docs/overview_and_guide.rst]: same as above

File: docs/overview_and_guide.rst [hunk 6]
```diff
@@ -44,6 +44,21 @@ Or you can use Homebrew to install the most up to date binary::
...
+
+      docker run -it --mount src=/PATH/TO/LOCAL/KEYSTORE,target=/keystore,type=bind raidennetwork/raiden:latest --keystore-path /keystore
+
+  Other flags such as the JSON-RPC endpoint to an Ethereum node can easily be chained to the command
```
Comment (czepluch) [inline docs/overview_and_guide.rst]: same as above

Comment (czepluch) [reply docs/overview_and_guide.rst]: And please add `.` at the end of the sentence.

File: docs/overview_and_guide.rst (hunk 2 above)
Comment (hackaugusto) [inline docs/overview_and_guide.rst]: Just a test
```suggestion
Docker
```

**Pull request merged**